examples/tv-app/linux/main.cpp

```diff
@@ -121,6 +120,6 @@ int main(int argc, char * argv[])
 
 void emberAfContentLauncherClusterInitCallback(EndpointId endpoint)
 {
-    ChipLogProgress(Zcl, "TV Linux App: ContentLauncherManager::SetDelegate");
-    chip::app::Clusters::ContentLauncher::SetDelegate(kContentLauncherEndpoint, &contentLauncherManager);
+    ChipLogProgress(Zcl, "TV Linux App: ContentLauncherManager::SetDefaultDelegate");
+    chip::app::Clusters::ContentLauncher::SetDefaultDelegate(&contentLauncherManager);
```

Comment:
Is there any conflict with the video player's contentLauncherManager?  Should we have an endpoint parameter for SetDefaultDelegate?

Reply:
For the tv, the content launcher may appear on endpoint 1, and then may appear on each content app endpoint. There is logic in content-launch-server to determine if there is an content app associated with the endpoint, so don't think that there is a need to assign a per-endpoint delegate here. Let me know if that makes sense.